src/main/java/com/ericsson/ei/subscription/SubscriptionHandler.java

```diff
@@ -96,17 +99,23 @@ private void extractConditions(final String aggregatedObject,
             final JsonNode subscriptionJson = new ObjectMapper().readTree(
                     subscriptionData);
             LOGGER.debug("SubscriptionJson : " + subscriptionJson.toString());
-            LOGGER.debug("Aggregated Object : " + aggregatedObject);
+            LOGGER.debug("Aggregated Object : " + aggregatedObject +" for the id: " + id);
             final ArrayNode requirementNode = (ArrayNode) subscriptionJson.get(
                     "requirements");
             LOGGER.debug("Requirements : " + requirementNode.toString());
-            final Iterator<JsonNode> requirementIterator = requirementNode.elements();
+
+            SubscriptionField subscriptionField = new SubscriptionField(subscriptionJson);
+            String subscriptionName = subscriptionField.get("subscriptionName");
+            final Iterator<JsonNode> requirementIterator = requirementNode.elements();            
             if (runSubscription.runSubscriptionOnObject(aggregatedObject,
                     requirementIterator, subscriptionJson, id)) {
                 LOGGER.debug(
                         "The subscription conditions match for the aggregatedObject");
                 informSubscriber.informSubscriber(aggregatedObject,
                         subscriptionJson);
+                LOGGER.info("Subscription with name ** {} ** has been processed for the event id: {}", subscriptionName, id);
```

Comment:
Same for these log statements, what id is it we're talking about here? event id? agrgegated object id?

Reply:
These log message that is for trace the execution in the logs should be logged as TRACE logs per definition.

Reply:
done